pynos/versions/base/interface.py

```diff
@@ -2926,3 +2925,251 @@ def get_port_chann_detail_request(last_aggregator_id):
 
     def bfd(self, **kwargs):
         raise NotImplementedError
+
```

Comment:
Why is this method added again? I thought we already have a method like enable_switchport or something similar